src/OpenApi/Model/Paths.php

```diff
@@ -29,6 +29,8 @@ public function getPath(string $path): ?PathItem
 
     public function getPaths(): array
     {
-        return $this->paths ?? [];
+        ksort($this->paths);
+
+        return $this->paths;
```

Comment:
This should be done in the OpenApiNormalizer https://github.com/api-platform/core/blob/main/src/OpenApi/Serializer/OpenApiNormalizer.php#L59 not here.

Reply:
@soyuka Thanks. I was thinking to do it in the Normalizer first, then thought of doing it in the origin of the data. Should I change it then or did you take over the task?

Reply:
thanks @stiffroy  I took the liberty of hanlding this myself!